src/mobile/apps/auth/test/client.test.coffee

```diff
@@ -98,7 +101,9 @@ describe 'Login view', ->
   beforeEach (done) ->
     global.location.search = ''
     benv.setup =>
-      benv.expose { $: benv.require 'jquery' }
+      benv.expose
+        $: benv.require 'jquery'
+        jQuery: benv.require 'jquery'
```

Comment:
Are these jquery changes related?

Reply:
These are more finicky tests that were exposed when the codecov changes went into place, maybe makes sense to PR separately? Had an easier time getting this running green with them.